GameEngine/Engine.cs

```diff
@@ -88,5 +86,27 @@ public static void Start(Func<IGameData> gameDataFactory, int loadFromSlot = 0)
                 }
             }
         }
+
+        public void StartNewGame()
+        {
+            string intro = gameData.GetGameIntroduction();
+            Console.Clear();
+            Console.WriteLine(intro);
+
+            Console.WriteLine($"First, we are going to set up your Character.");
+            Console.WriteLine($"Now, you are {gameData.PlayerName}, do you want to change it? Yes/No");
+
+            if(Console.ReadLine().Equals("yes", StringComparison.OrdinalIgnoreCase))
+            {
+                string PlayersNewName;
```

Comment:
This is a local variable ( a variable created inside of a method), please always name these starting with a lowercase character.